app/controllers/v1/gids/lce/certifications_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module V1
+  module GIDS
+    module Lce
+      class CertificationsController < LceController
+        def show
+          render json: service.get_certification_details_v1(scrubbed_params)
```

Comment:
Is it possible to stub out tests for the 5 controller files you are adding here?

Reply:
@rmtolmach yes can do. One question. It looks like for the existing gids controllers (yellow ribbon, institutions, etc), there aren't any spec tests for the controllers. Instead I see request specs (at least for v0 requests... I don't see anything for v1 controllers). I put a ticket on our backlog to write request specs but figured I would wait until the real data is flowing from gids. Does the platform have a preference for controller specs vs request specs?

Reply:
oh, my bad, yeah request specs are the way!